model/delivery/LtiTestTakerAuthorizationService.php

```diff
@@ -70,6 +72,28 @@ public function isProctored($deliveryId, User $user)
         }
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function verifyResumeAuthorization(DeliveryExecutionInterface $deliveryExecution, User $user)
+    {
+        parent::verifyResumeAuthorization($deliveryExecution, $user);
+
+        if ($this->isAuthorizedLtiExecutionWithWrongUrl($deliveryExecution)) {
+            throw new UnAuthorizedException(
```

Comment:
Shouldn't we have a similar redirect then in case of a regular non-LTI-proctored assessment?

Reply:
Maybe, I did not check this case (I think it wont happen - cause URL gets mixed only by LTI launch), but since this became a blocker for February release and I need to backport it, I've focused in this case in isolation.

Reply:
OK, let's keep that case in mind as well.
I guess we can ask a QA engineer to validate the case while verifying your fix+backport.

Could you please mention that case when you'll be passing the ticket over to the QA department?

Reply:
Sure I can follow this question. Would we need to create a new env for this or do you have one we can check?

I've deployed the fix to https://demo.selor.taocloud.org